src/number_theory/class_number/finite.lean

```diff
@@ -392,12 +392,16 @@ begin
   let bS := b.map ((linear_map.quot_ker_equiv_range _).symm.trans _),
```

Comment:
The underscore here spawns off lots of unwanted side goals (Lean can't tell we just want a plain linear map, not a semilinear one).

I've patched this for now, see below, but it is an ugly solution. 

@dupuisf, could you see if you can do something better?

Reply:
Is using the trans notation instead enough?

Reply:
Oops, I went and fixed this before I saw all the new comments. Yes using the new trans notation is enough.